src/main/java/uk/co/ramp/covid/simulation/Model.java

```
@@ -160,13 +161,24 @@ public boolean isValid() {
         return valid;
     }
 
+    /** Runs the model with the given parameters. Returns null if the parameters are missing or invalid */
     public List<List<DailyStats>> run() {
-        assert isValid() : "Model parameters are invalid";
+        if (!isValid()) {
+            LOGGER.error("Model parameters missing or invalid");
+            return null;
```

I think this should throw an exception here.